src/views/Home.vue

```diff
@@ -9,6 +9,7 @@
 import Start from "@/components/Start.vue";
 
 export default {
+  //Why is this name "Resources"? Shouldn't it be "Home"? I'm confused
```

Comment:
Remove this comment (as it doesn't add anything) or update the name to Home and we're good.

Reply:
Okay, thank you! I changed it on the file on my computer, then pushed up the change, so it's asking you to review again. 
I'm learning a lot in this process, and I really appreciate your patience as I figure this all out!